test/format/format_skip_errors.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+
+# Conditional grep
+grep -q "cache clean check: no" RUNDIR.*.log && grep -q "cache dirty check: no" RUNDIR.*.log && grep "format run more than 15 minutes past the maximum time" RUNDIR.*.log
+
+if [ $? -eq 0 ]
+then
+	echo "{cache clean check: no} && {cache dirty check: no} && {format run more than 15 minutes past the maximum time} Error found in the failed log"
+	exit 0
+fi
```

Comment:
I guess the reason this error checking odds out is it has more than one signatures to match? Can we use a multi-dimensional array (for the skip list) to solve the problem?

Reply:
WT-5489 was merged yesterday. It's likely we don't need to include this known error. While it's still useful to support multi-signature matching for a single known error.